tests/integration/kubernetes/gha-run.sh

```diff
@@ -265,6 +265,10 @@ function uninstall_kbs_client() {
 }
 
 function run_tests() {
+	if [ "${K8S_TEST_HOST_TYPE}" = "baremetal" ]; then
```

Comment:
It would be nice to explain here why this is useful for bare metal, but not for Host VMs.